assets/src/edit-story/karma/fixture/fixture.js

```diff
@@ -291,6 +291,14 @@ export class Fixture {
       }
     });
 
+    // Check to see if Roboto font is loaded.
+    await waitFor(async () => {
+      await document.fonts.load('0 Roboto', '');
+      if (!document.fonts.check(' 0 Roboto', '')) {
```

Comment:
Just to confirm, is `0 Roboto` the right syntax? I haven't tested it in the browserr when I wrote that comment.

Also, there's an errant leading space.

Reply:
```suggestion
      const font = '12px Roboto';
      await document.fonts.load(font);
      if (!document.fonts.check(font)) {
```

I don't think we need the space or the empty string. We could try and check to see if a special character here is loaded, like this, 

```
document.fonts.check("12px Roboto", "ß");
```

But this maybe overkill. 

You may also notice that I changed it to 12px from 0. Why not render a reasonable size to see if it works.

Reply:
Cool. Let's do `'12px Roboto', ''` and `'12px "Google Sans"', '')` then. (note the quotes for `Google Sans`)